packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -31,21 +31,21 @@ describe("wrangler", () => {
 			"wrangler
 
 			Commands:
-			  wrangler init [name]       📥 Create a wrangler.toml configuration file
-			  wrangler dev [script]      👂 Start a local server for developing your worker
-			  wrangler publish [script]  🆙 Publish your Worker to Cloudflare.
-			  wrangler tail [worker]     🦚 Starts a log tailing session for a published Worker.
-			  wrangler secret            🤫 Generate a secret that can be referenced in a Worker
-			  wrangler kv:namespace      🗂️  Interact with your Workers KV Namespaces
-			  wrangler kv:key            🔑 Individually manage Workers KV key-value pairs
-			  wrangler kv:bulk           💪 Interact with multiple Workers KV key-value pairs at once
-			  wrangler pages             ⚡️ Configure Cloudflare Pages
-			  wrangler r2                📦 Interact with an R2 store
-			  wrangler worker-namespace  📦 Interact with a worker namespace
-			  wrangler pubsub            📮 Interact and manage Pub/Sub Brokers
-			  wrangler login             🔓 Login to Cloudflare
-			  wrangler logout            🚪 Logout from Cloudflare
-			  wrangler whoami            🕵️  Retrieve your user info and test your auth config
+			  wrangler init [name]         📥 Create a wrangler.toml configuration file
+			  wrangler dev [script]        👂 Start a local server for developing your worker
+			  wrangler publish [script]    🆙 Publish your Worker to Cloudflare.
+			  wrangler tail [worker]       🦚 Starts a log tailing session for a published Worker.
+			  wrangler secret              🤫 Generate a secret that can be referenced in a Worker
+			  wrangler kv:namespace        🗂️  Interact with your Workers KV Namespaces
+			  wrangler kv:key              🔑 Individually manage Workers KV key-value pairs
+			  wrangler kv:bulk             💪 Interact with multiple Workers KV key-value pairs at once
+			  wrangler pages               ⚡️ Configure Cloudflare Pages
+			  wrangler r2                  📦 Interact with an R2 store
+			  wrangler dispatch-namespace  📦 Interact with a dispatch namespace
```

Comment:
Is this intentionally vague?

Reply:
It's an experimental feature, so it's not super documented or widely available yet. I suspect this will change a little in the future, so it's fine for now?